game_runner.rb

```diff
@@ -0,0 +1,19 @@
+class GameRunner
+  def initialize(stdout)
+    @stdout = stdout
+  end
+
+  def run
```

Comment:
Now that you have some working code. Do another branch that is for refactors. 

Look at each of the rules and think about whether it is applicable, and if so how to meet it.